coffea/lumi_tools/lumi_tools.py

```diff
@@ -117,6 +118,10 @@ def __call__(self, runs, lumis):
                 An array of dtype `bool` where valid (run, lumi) tuples
                 will have their corresponding entry set ``True``.
         """
+        if isinstance(runs, ak.highlevel.Array):
```

Comment:
FYI for future: `numpy.asarray` can be used as well, avoiding the awkward import (and generalizing to other array extension types)